internal/proto/controller/api/resources/roles/v1/role.proto

```diff
@@ -91,13 +91,17 @@ message Role {
   // The Scope the grants will apply to. If the Role is at the global scope, this can be an org or project. If the Role is at an org scope, this can be a project within the org. It is invalid for this to be anything other than the Role's scope when the Role's scope is a project.
   google.protobuf.StringValue grant_scope_id = 90 [
     json_name = "grant_scope_id",
-    (custom_options.v1.generate_sdk_option) = true,
-    (custom_options.v1.mask_mapping) = {
-      this: "grant_scope_id"
-      that: "GrantScopeId"
-    }
+    deprecated = true,
+    (custom_options.v1.generate_sdk_option) = true
   ]; // @gotags: `class:"public" eventstream:"observation"`
 
+  // Output only. The IDs of Scopes the grants will apply to. This can include
+  // the role's own scope ID, or "this" for the same behavior; specific IDs of
+  // scopes that are children of the role's scope; the value "children" to match
+  // all direct child scopes of the role's scope; or the value "descendants" to
+  // match all descendant scopes (e.g. child scopes, children of child scopes).
+  repeated string grant_scope_ids = 91 [json_name = "grant_scope_ids"]; // @gotags: `class:"public"`
```

Comment:
The answer is no to both, because they are not the role's own ID (/ "this") or children of the role's scope, or match the conditions for the special values.